gradio/cli/commands/components/docs.py

```diff
@@ -0,0 +1,101 @@
+import importlib
+from pathlib import Path
+from typing import Optional
+
+import requests
+import toml
+from typer import Argument, Option
+from typing_extensions import Annotated
+
+from ._docs_utils import extract_docstrings, get_deep, make_markdown, make_space
```

Comment:
I ran the docs command for all of the components - I only got two failures

- `Fallback` component (e.g. no template) - fails with `KeyError: 'return'`
- `Plot` component - fails with `KeyError: 'value'`

Haven't checked the output yet. That's next.

Reply:
Ah `Fallback` has no hints or docstrings. I'll see if we can fail more gracefully or throw an error if the output isn't helpful.

Strange behaviour for `Plot`, i think it has everything is should. Will check that.